### PR TITLE
Henry/front end tabs

### PR DESCRIPTION
**Pull request by henrytran28**
image

Package-lock.json is sorta big :( and according to this we need to keep it
https://stackoverflow.com/questions/44206782

### CODE REVIEW

**Comment by henrytran28** [comment]
Can I update travis to install node 8.9+?

File: web-app/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import { ControlsComponent } from './tabs/controls/controls.component';
+import { MotorComponent } from './tabs/motor/motor.component';
+import { MpptComponent } from './tabs/mppt/mppt.component';
+import { FaultsComponent } from './tabs/faults/faults.component';
+
+const routes: Routes = [
+    {
+        path: '',
+        redirectTo: '/battery',
+        pathMatch: 'full'
```
Comment (bill-luu) [inline web-app/src/app/app-routing.module.ts]: The formatting on this path should match the others.

Comment (bill-luu) [reply web-app/src/app/app-routing.module.ts]: Also, is this needed? I thought that ** would do the same thing as well?

**Comment by nguyen-tran** [comment]
I've created a pull request where I vectored the icons you guys are using here: https://github.com/UCSolarCarTeam/Epsilon-Telemetry-Server/pull/7

**Pull request merged**